libs/wingc/src/lib.rs

```diff
@@ -199,7 +199,9 @@ pub fn type_check(
 			parameters: vec![types.bool()],
 			return_type: types.void(),
 			phase: Phase::Independent,
-			js_override: Some("{((cond) => {if (!cond) throw new Error(`assertion failed: '$args$'`)})($args$)}".to_string()),
+			js_override: Some(
+				"{((cond) => {if (!cond) throw new Error('assertion failed: \"$args$\"')})($args$)}".to_string()
+			),
```

Comment:
Try this perhaps?
```suggestion
			js_override: Some(
				r#"{((cond) => {if (!cond) throw new Error("assertion failed: \"" + $args$ + "\"")})($args$)}"#.to_string()
			),
```

Reply:
I think we need to escape the internal string.

A nice trick is to use `JSON.stringify()`. It will return a quoted string that you can just plug into the other string.

Reply:
Thanks guys! We have tried both of those options (JSON.stringify and raw strings) but they seem to break some other tests. Ive been pairing with @hasanaburayyan on a fix for this. We are still trying to come up with a robust solution for this.